test/deadline_submitter_for_houdini/unit/test_assets.py

```diff
@@ -90,3 +92,79 @@ def test_get_recursive_output_directories(node_type: str, output_parm_name: str)
     node.parm.assert_called_once_with(output_parm_name)
     inner_node.parm.assert_called_with("vm_picture")
     assert out_dirs == {"/test/output/directory/mantra"}
+
+
+@pytest.mark.parametrize(
+    "scene_detected_assets, current_assets, expected_input_filenames, expected_input_directories, expected_output_directories",
+    [
+        (AssetReferences(), AssetReferences(), [], [], []),
```

Comment:
can we leverage `pytest.param` here and provide `ids` to all of these test cases? Should make it easier to identify the intent of the test, and understand when things go wrong with the tests. Should also help idenitify gaps in the testing that we have for it. ie.

```
[ 
    pytest.param(AssetReferences(), AssetReferences(), [], [], [], id="no assets"),, 
    pytest.param(..., id="only auto detected inputs"),
    ...
]
```